xml/libvirt_overview.xml

```diff
@@ -4,80 +4,487 @@
 
- <important>
-  <title>Conflicting services: <systemitem class="daemon">libvirtd</systemitem>
-  and <systemitem class="daemon">xendomains</systemitem></title>
-  <para>
-   If <systemitem class="daemon">libvirtd</systemitem> fails to start,
-   check if the service <systemitem class="daemon">xendomains</systemitem> is
-   loaded:
-  </para>
-  <screen>&prompt.user;systemctl is-active xendomains
-active</screen>
-  <para>
-   If the command returns <literal>active</literal>, you need to stop
-   <systemitem class="daemon">xendomains</systemitem> before you can
-   start the <systemitem class="daemon">libvirtd</systemitem> daemon. If
-   you want <systemitem class="daemon">libvirtd</systemitem> to also start
-   after rebooting, additionally prevent <systemitem
-   class="daemon">xendomains</systemitem> from starting automatically. Disable
-   the service:
-  </para>
-  <screen>&prompt.sudo;systemctl stop xendomains
+    <important>
+      <title>Conflicting services: <systemitem class="daemon">libvirtd</systemitem> and <systemitem class="daemon">xendomains</systemitem></title>
+      <para>
+        If &libvirtd; fails to start, check if the service
+        <systemitem class="daemon">xendomains</systemitem> is loaded:
+      </para>
+<screen>&prompt.user;systemctl is-active xendomains active</screen>
+      <para>
+        If the command returns <literal>active</literal>, you need to stop
+        <systemitem class="daemon">xendomains</systemitem> before you can start
+        the &libvirtd; daemon. If you want &libvirtd; to also start after
+        rebooting, additionally prevent
+        <systemitem
+    class="daemon">xendomains</systemitem> from starting
+        automatically. Disable the service:
+      </para>
+<screen>
+&prompt.sudo;systemctl stop xendomains
 &prompt.sudo;systemctl disable xendomains
-&prompt.sudo;systemctl start libvirtd</screen>
-  <para>
-   <systemitem class="daemon">xendomains</systemitem> and <systemitem
-   class="daemon">libvirtd</systemitem> provide the same service and when used
-   in parallel may interfere with one another. As an example, <systemitem
-   class="daemon">xendomains</systemitem> may attempt to start a domU already
-   started by <systemitem class="daemon">libvirtd</systemitem>.
-  </para>
- </important>
+&prompt.sudo;systemctl start libvirtd
+</screen>
+      <para>
+        <systemitem class="daemon">xendomains</systemitem> and &libvirtd;
+        provide the same service and when used in parallel may interfere with
+        one another. As an example,
+        <systemitem class="daemon">xendomains</systemitem> may attempt to start
+        a domU already started by &libvirtd;.
+      </para>
+    </important>
+  </sect1>
+  <sect1 xml:id="libvirt-modular-daemon">
+    <title>Starting and stopping the modular daemons</title>
+
+    <para>
+      The modular daemons are named after the driver which they are running,
+      with the pattern 'virt<replaceable>DRIVER</replaceable>d'. They are
```

Comment:
Can we use `literal` or `quote` element for 'virt<replaceable>DRIVER</replaceable>d' ?

Reply:
yep